mne/epochs.py

```diff
@@ -2132,9 +2132,18 @@ def _read_one_epoch_file(f, tree, fname, preload):
         if len(processed) == 0:
             raise ValueError('Could not find processed data')
 
-        epochs_node = dir_tree_find(tree, FIFF.FIFFB_EPOCHS)
+        if fix_tags:
+            epochs_node = dir_tree_find(tree, 122)
+        else:
+            epochs_node = dir_tree_find(tree, FIFF.FIFFB_EPOCHS)
         if len(epochs_node) == 0:
-            raise ValueError('Could not find epochs data')
+            if fix_tags:
+                msg = 'Could not find epochs data'
+            else:
+                msg = ('Could not find epochs data. If the data is saved with '
+                       'old tagging system, the data can be read by using '
+                       'fix_tags option.')
+            raise ValueError(msg)
```

Comment:
I don't think this is what we want to do. Users don't need to care about fixing the tags. We should just look for either tag, but from now on, save only with the correct one. Make sense?